WordPress/src/main/java/org/wordpress/android/models/Role.java

```diff
@@ -21,21 +27,42 @@
         this.backgroundColor = backgroundColor;
     }
 
-    public static String toString(Context context, Role role) {
+    // Label to be used in UI
+    public static String getLabel(Context context, Role role) {
         return context.getString(role.label);
     }
 
-    // This method will be used to determine the role of the user from network request
-    public static Role fromString(String value) {
+    // Returns the key for that role
+    public static String toKey(Role role) {
+        switch (role) {
+            case SUPER_ADMIN:
+                return KEY_SUPER_ADMIN;
+            case ADMIN:
+                return KEY_ADMIN;
+            case EDITOR:
+                return KEY_EDITOR;
+            case AUTHOR:
+                return KEY_AUTHOR;
+            case CONTRIBUTOR:
+                return KEY_CONTRIBUTOR;
+            default:
```

Comment:
Apparently the `UNSUPPORTED` case is missing from the switch. It gets caught by the `default` case.

If this is intentional, I'd argue that it's better to be specific in handling `UNSUPPORTED` since it is defined just like the other enum instances.

In any case, I would argue that in this Enum -> String conversion, the `default` case offers no value as there are no other or "unknown" cases that is unpractical to enumerate and we'd prefer to group under the `default` case. Also, the `default` will help hide any future bugs when new cases might get added to the enum.